lib/index.js

```diff
@@ -49,12 +49,12 @@ module.exports.getRecipes = recipes.getAllRecipes;
  * @param {Object|String} NEEOBrain to connect or an IP/hostname to connect
  * @return {Promise} promise contains an array with powerKey's of the powered on recipes.
  * @example
- * neeoapi.getRecipesPowerState(brain)
+ * neeoapi.getActiveRecipes(brain)
  * .then((poweredOnKeys) => {
  *   console.log('- Power state fetched, powered on recipes:', poweredOnKeys);
  * })
  */
-module.exports.getRecipesPowerState = recipes.getRecipePowerState;
+module.exports.getActiveRecipes = recipes.getActiveRecipes;
```

Comment:
thanks for your work so far! last thing: as we don't want to break existing implementations right now, would you mind adding back the old one that points to the new function? Then existing implementations won't break

```
// dont break the SDK, @deprecated
module.exports.getRecipesPowerState = recipes.getActiveRecipes;
```